wlauto/external/uiauto/src/com/arm/wlauto/uiauto/BaseUiAutomation.java

```diff
@@ -43,6 +43,15 @@
 import com.android.uiautomator.testrunner.UiAutomatorTestCase;
 
 public class BaseUiAutomation extends UiAutomatorTestCase {
+    public Bundle parameters;
+    public String packageName;
+    public String packageID;
+
+    //Get application package parameters and create package ID
+    public void getPackageParameters() {
```

Comment:
moved to UxPerfUiautomation() class as all uxperf workloads have this. It can be cleaned further when we work with cleaning our classes later.